### PR TITLE
Scaffold

### PR DESCRIPTION
**Pull request by emileswarts**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+source 'http://rubygems.org'
+ruby File.read('.ruby-version').chomp
```
Comment (naliwajek) [inline Gemfile]: Very nice idea 👍

File: mysql/bin/wait_for_mysql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# The Jenkins runner doesn't support TTY sessions meaning docker-compose exec commands
+# failed to execute properly and would hang the CI
+# https://github.com/docker/compose/issues/5696
+until docker-compose exec -T db mysql -hdb -uroot -proot -e 'SELECT 1' &> /dev/null
+do
+  printf "."
+  sleep 1
+done
```
Comment (naliwajek) [inline mysql/bin/wait_for_mysql]: Has this been working for you? I remember we had problems getting this to be consistent.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+lint:
+	$(MAKE) build
+	docker-compose run --rm app bundle exec govuk-lint-ruby
+
+test:
+	$(MAKE) serve
+	./mysql/bin/wait_for_mysql
+	docker-compose run --rm app rspec
+	$(MAKE) stop
```
Comment (naliwajek) [inline Makefile]: Is there a reason why we don't use a watch, e.g `guard` gem or similar, on those tests?

**Comment by emileswarts** [comment]
Agreed using guard would be nice.  Current docker setup is quite slow as it boots up everything from a clean slate every time.  Needs looking into.

File: Gemfile (hunk 1 above)
Comment (emileswarts) [reply Gemfile]: Thanks.  Not my idea though

File: mysql/bin/wait_for_mysql (hunk 2 above)
Comment (emileswarts) [reply mysql/bin/wait_for_mysql]: Yup this script works fine for this

File: Makefile (hunk 3 above)
Comment (emileswarts) [reply Makefile]: Good idea, something to look into

**Pull request merged**